guides/updating-from-cdflow-classic.md

```diff
@@ -11,36 +11,44 @@ We have updated the implementation of cdflow and made some changes to the projec
 ## Step-by-step
 
 1. You'll need the `infra` folder in your repo with the new `.tf` files.
+   You can copy the files from the [node-minimal-boilerplate](https://github.com/mergermarket/node-minimal-boilerplate/tree/master/infra) repo.
 
-    You can copy the files from the [node-minimal-boilerplate](https://github.com/mergermarket/node-minimal-boilerplate/tree/master/infra) repo.
-2. Add a `cdflow.yml` file. Example below:
-    ```yaml
-    account-scheme-url: s3://mmg-account-resources/account-scheme.json
-    type: docker
-    team: DevTeam-Platform
-    ```
+2. Add a `cdflow.yml` file. 
+Please see the [cdflow.yml reference](/reference/cdflow-yaml) for more detail.
+Example below:
+```yaml
+account-scheme-url: s3://mmg-account-resources/account-scheme.json
+type: docker
+team: DevTeam-Platform
+```
 3. Update the `Jenkinsfile` to use `cdflow` from PATH
```

Comment:
Is it worth to mention here, that we also support normal Jenkins jobs (old-style pipeline configuration) or make it more generic (Update your CI to use `cdflow` from `PATH`?)